### ISSUE TITLE
 Returning the result of insert/update ops

### ISSUE THREAD

**Issue by encephalone** [Flowneee/tarantool-rs#7]
Implementation example:

```
    async fn insert_with_result<T, R: DeserializeOwned>(&self, space_id: u32, tuple: T) -> Result<R>
    where
        T: Tuple + Send,
    {
        let body = self.send_request(Insert::new(space_id, tuple)).await?;
        extract_and_deserialize_iproto_data(body).map_err(Into::into)
    }
   ```

**Comment by Flowneee** [comment]
Implemented in `0.0.8`. Usage example here https://github.com/Flowneee/tarantool-rs/blob/v0.0.8/tests/integration.rs#L218.

**Comment by encephalone** [comment]
Great! Thank you
